backend/routes/admin.js

```diff
@@ -95,10 +95,29 @@ async function createUser(user) {
 
 router.post("/create-user", async function(req, res) {
   try {
-    const user = await createUser(req.body);
-    res.json(user);
+    const userRecord = await createUser(req.body);
+    // Insert record into mongodb
+    const mongoUserRecord = {
+      userId: userRecord.uid,
+      name: req.body.name,
+      preferredName: req.body.preferredName,
+      email: userRecord.email,
+      role: "User",
+      programs: req.body.programs
+    };
+    try {
+      await db.users.updateOne({ userId: req.body.userId }, mongoUserRecord, {
+        upsert: true
```

Comment:
Yeah I agree ideally we should be doing an insertOne, I originally left it as updateOne bc I wasn't 100% confident in our mapping of users from firebase to mongo, so I wanted to avoid creating duplicate users in mongo if possible.
After thinking about it a bit more though, I think it's just better to use insertOne withwith _id matching uid from firebase, so that if there's already a user then it'll just throw an error. LMK if that sounds good to you!

Reply:
I like error vs no error

Reply:
Good point, I hadn't considered the possibility of existing users. In that case maybe we should revert back to upsert, I don't think using `insertOne` and checking with `_id` works because `_id` is an auto-generated value for all previous records and they don't match uids. Sorry for the back and forth on this!

Reply:
Yeah I think going back to upsert is best... can't use uids anyway because they dont match the required length for object ids :(